tests/functional/otlp-metrics/00-install-jaeger.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: otel-metrics
+
+---
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger-production
+  namespace: otel-metrics
+spec:
+  strategy: production
+  ingress:
+    security: none
+  storage:
+    type: elasticsearch
+    elasticsearch:
```

Comment:
This will only work in OpenShit using the ElasticSearch OpenShift Operator

Reply:
Use an allinone for the Jaeger deployment strategy.